services/rsources/stats_collector_test.go

```diff
@@ -335,7 +426,7 @@ var _ = Describe("Using StatsCollector", Serial, func() {
 					err = recover().(error)
 					Expect(err).ToNot(BeNil())
 				}()
-				statsCollector.JobStatusesUpdated(jobStatuses)
+				statsCollector.CollectStats(jobStatuses)
```

Comment:
please change all text literal occurences of `JobStatusesUpdated` with `CollectStats`